pages/About.tsx

```diff
@@ -0,0 +1,96 @@
+import type { NextPage } from 'next'
+import styles from '../styles/About.module.css'
+import Box from '@mui/material/Box'
+import {Button} from '@mui/material'
+import Card from '@mui/material/Card';
+import CardActions from '@mui/material/CardActions';
+import CardContent from '@mui/material/CardContent';
+import CardHeader from '@mui/material/CardHeader';
+import AppBar from "../components/AppBar";
+import Typography from '@mui/material/Typography';
+import React, {Component} from "react";
+import TextField from '@mui/material/TextField'
+import TextareaAutosize from '@mui/material/TextareaAutosize'
+
+import NewspaperIcon from '@mui/icons-material/Newspaper';
+import GroupsIcon from '@mui/icons-material/Groups';
+import ReceiptLongIcon from '@mui/icons-material/ReceiptLong';
+import FindInPageIcon from '@mui/icons-material/FindInPage';
+import SendIcon from '@mui/icons-material/Send';
+
+const cards = [
+    {component: NewspaperIcon, name: "Publication", content: "Partagez vos articles, choisissez comment vous voulez les partager."},
+    {component: GroupsIcon, name: "Mise en relation", content: "Interactivité entre les acteurs du monde de la recherche."},
+    {component: ReceiptLongIcon, name: "Relecture Anonyme", content: "Relecture en ligne, anonyme jusqu'à publication, pour garantir l'objectivité des avis de celle-ci."},
+    {component: FindInPageIcon, name: "Consultations", content: "Consulter des papiers de recherches gratuites ou payantes."}
+]
+
+const About: NextPage = () => {
+    return (
+        <main className={styles.main}>
+            <Box component="div" className={styles.aboutHeroContainer}>
+                <Box component="p" className={styles.aboutHeroTitle}>ResearchShare - Knowledge for everyone</Box>
+                <Box component="p" className={styles.aboutHeroDescription}>Une plateforme web de recherche scientifique, dédiée aux chercheurs, avec une volonté d’apporter plus d'interactivité entre acteurs du monde de la recherche.</Box>
```

Comment:
```suggestion
                <Box component="p" className={styles.aboutHeroDescription}>Une plateforme web de recherche scientifique, dédiée aux chercheurs, avec une volonté d&apos;apporter plus d&apos;interactivité entre acteurs du monde de la recherche.</Box>
```
You need to replace `'` with `&apos;`